src/PkgDeps.jl

```diff
@@ -129,4 +152,28 @@ function find_downstream_dependencies(
     return downstream_dependencies
 end
 
+
+"""
+    users(pkg_name::String; pkg_registry_name::String=GENERAL_REGISTRY)
```

Comment:
```suggestion
    users(pkg_name::String; pkg_registry_name::String=$GENERAL_REGISTRY)
```

Reply:
This doesn't work?

Reply:
Really? What's the errors?

Reply:
```julia
julia> const A = "asdf"
"asdf"

julia> f(; var::String=$A) = println(var)
ERROR: syntax: "$" expression outside quote around REPL[2]:1
Stacktrace:
 [1] top-level scope
   @ REPL[2]:1
```

Reply:
this is in the docstring

Reply:
although since this is a string in a strin, i guess it should be
```suggestion
    users(pkg_name::String; pkg_registry_name::String="$GENERAL_REGISTRY")
```

Reply:
Lol, good point, I completely missed that. I'll adjust it!